### PR TITLE
Api additions - Get Networks

### PR DESCRIPTION
**Pull request by therealsmartalec**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [ ] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [ ] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [ ] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [ ] My PR has been functionally tested.
- [ ] All of the [unit-tests](http://forsetisecurity.org/docs/development/#executing-tests) still pass.
- [ ] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: tests/common/gcp_api/test_data/fake_compute_responses.py [hunk 1]
```diff
@@ -1083,3 +1083,73 @@
...
+NETWORKS_LIST_PAGE2 = """
+{
+ "kind": "compute#networkList",
+ "id": "projects/project1/global/networks",
+ "items": [
+  {
+   "kind": "compute#network",
+   "id": "1234",
+   "creationTimestamp": "2017-09-25T12:41:09.416-07:00",
+   "name": "default2",
```
Comment (ahoying) [inline tests/common/gcp_api/test_data/fake_compute_responses.py]: nit: name should match selflink, change from default2 to thisisatest and update the expected names below.

File: google/cloud/security/common/gcp_api/compute.py [hunk 2]
```diff
@@ -194,6 +195,16 @@ def firewalls(self):
         return self._firewalls
 
     @property
+    def networks(self):
+        """Returns a _ComputeNetworksRepository instance."""
+        # The beta api provides more complete gcp networks data.
```
Comment (ahoying) [inline google/cloud/security/common/gcp_api/compute.py]: nit: Comment is out of date, no longer needed with the change back to v1.

File: tests/common/gcp_api/compute_test.py [hunk 3]
```diff
@@ -117,6 +117,17 @@ def test_get_firewall_rules_errors(self, name, response, status,
         with self.assertRaises(expected_exception):
             list(self.gce_api_client.get_firewall_rules(self.project_id))
 
+    def test_get_networks(self):
```
Comment (ahoying) [inline tests/common/gcp_api/compute_test.py]: Please add a test for the error cases as well, like test_get_firewall_rules_errors above.

Comment (therealsmartalec) [reply tests/common/gcp_api/compute_test.py]: Will do!

**Pull request merged**